package.json

```diff
@@ -9,7 +9,7 @@
     }
   },
   "dependencies": {
-    "@openzeppelin/contracts": "^4.5.0",
```

Comment:
Esse arquivo não precisa ser alteado! Podes dar um rollback nele pfv?

Quando for fazer o commit pode descartar alterações que não devem ser commitas